etl/steps/data/garden/health/2024-04-22/gpei_funding.py

```diff
@@ -0,0 +1,43 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow = paths.load_dataset("gpei_funding")
+
+    # Read table from meadow dataset.
+    tb = ds_meadow["gpei_funding"].reset_index()
+
+    #
+    # Process data.
+    tb["total"] = tb[
+        [
+            "domestic_resources",
```

Comment:
You can maybe keep this list as a variable at the top and assert if the dataset has those columns (for future updates)